cishouseholds/extract.py

```diff
@@ -35,8 +35,11 @@ def list_contents(
     for line in ls.stdout:  # type: ignore
         dic = {}
         f = line.decode("utf-8")
+        attributes = f.split()
+
+        attributes = [*attributes[:7], " ".join(attributes[7:])]
```

Comment:
Why is part of this being reversed? This function is used for all of the input processing, so I'm keen not to change it unless necessary

Reply:
most of the soc files are filled with spaces and the previous method did not account for this. this correction accounts for this and will not affect the current processing